app/models/user_role.rb

```diff
@@ -30,8 +30,16 @@ def self.facility_management_roles
     facility_roles - [FACILITY_STAFF, FACILITY_SENIOR_STAFF]
   end
 
+  def self.global_roles
+    account_manager + administrator + billing_administrator
+  end
+
   def self.valid_roles
-    account_manager + administrator + billing_administrator + facility_roles
+    global_roles + facility_roles
+  end
+
+  def self.global
+    where(facility_id: nil)
```

Comment:
I feel like users with `Administrator` with a non-NULL `facility_id` should either become Facility Administrators or Global Administrators. The users who are also Billing Administrators (with no facility) I'm guessing were probably intended to be Global Administrators, at least. There are only a few so maybe we can get clarification from UIC on them.